src/components/SelectInput/SelectInput.tsx

```diff
@@ -159,6 +159,7 @@ export interface SelectOption {
   title: string;
   value: any;
   iconUrl?: string;
+  gasCostUSD?: string;
```

Comment:
not sure it's reusable as select input can be ANYTHING, without PRICE, we should rename this as `custom/customAddon` or similar or reuse `title` for this

Reply:
@MayukhSamadder2000

Reply:
@poocart you mean renaming the `gasCostUSD`  on `SelectInput` to custom? in that case custom can be anything as you said isn't it better to use optional gascostusd? As anyways it'll be an optional dependency?